tests/actions/test.subs-actions.js

```diff
@@ -70,47 +70,34 @@ describe('subscription actions', function() {
 
   describe('createSubscription', function() {
 
+    var fakeFetch;
     var fakeCard = {
       number: '411111111111',
       cvv: '222',
       expiration: '06/16',
     };
 
-    // Bad card with missing cvv
-    // and expiration.
-    var fakeBadCard = {
-      number: '411111111111',
-    };
-
-    class FakeBraintreeClient {
-      tokenizeCard(config, callback) {
-        console.log('resolving braintree tokenization');
-        callback(null, 'some-nonce');
-      }
-    }
-
-    class FakeBraintreeClientError {
-      tokenizeCard(config, callback) {
-        console.log('resolving braintree tokenization with error');
-        callback("I'm some error");
-      }
-    }
+    beforeEach(function() {
+      dispatchSpy = sinon.spy();
+      fakeFetch = helpers.fakeFetch();
+    });
 
-    function createSubscription(fetch, client=FakeBraintreeClient,
-                                payMethod=fakeCard) {
-      var deferredAction = subActions.createSubscription(
-                                          'product-id',
-                                          payMethod,
-                                          'braintree-token',
-                                          fetch,
-                                          client);
+    function createSubscription({fetch=fakeFetch,
+                                 client=helpers.FakeBraintreeClient,
+                                 payMethod=fakeCard} = {}) {
```

Comment:
Thanks for the link, thats what i was looking for, specifically the {...} = {}. Yay for learning in pull requests.